src/psycop_feature_generation/loaders/raw/load_medications.py

```diff
@@ -40,13 +40,20 @@ def load(
             atc_code.
         n_rows (int, optional): Number of rows to return. Defaults to None, in which case all rows are returned.
         exclude_atc_codes (list[str], optional): Drop rows if atc_code is a direct match to any of these. Defaults to None.
-        administration_route (str, optional): Whether to subset by a specific administration route, e.g. 'OR', 'IM' or 'IV'. Defaults to None.
-        administration_method (str, optional): Whether to subset by method of administration, e.g. 'PN' or 'Fast'. Defaults to None.
+        administration_route (str, optional): Whether to subset by a specific administration route, e.g. 'OR', 'IM' or 'IV'. Only applicable for administered medication, not prescribed. Defaults to None.
+        administration_method (str, optional): Whether to subset by method of administration, e.g. 'PN' or 'Fast'. Only applicable for administered medication, not prescribed. Defaults to None.
 
     Returns:
         pd.DataFrame: Cols: dw_ek_borger, timestamp, {atc_code_prefix}_value = 1
     """
 
+    if load_prescribed == True and (
+        administration_method is not None or administration_route is not None
+    ):
```

Comment:
```suggestion
    if load_prescribed and any([administration_method, administration_route]):
```

Tiny tiny thing, just makes it more pythonic and brief :-)